react/components/EXPERIMENTAL_TableTree/Tree/CellPrefix.tsx

```diff
@@ -1,27 +1,27 @@
-import React, { FC, Children } from 'react'
+import React, { FC } from 'react'
 
 import CollapseToggle, { CollapseToggleProps } from './CollapseToggle'
 import Checkbox, { CheckboxProps } from '../../EXPERIMENTAL_Table/Checkbox'
 
-const PREFIX_WIDTH = 48
-const CHECKBOXES_WIDTH = 16
+const GAP = 35
 
 const CellPrefix: FC<CellPrefixProps> & CellPrefixComposites = ({
   children,
-  hasCheckbox,
   depth,
 }) => {
-  const width = (PREFIX_WIDTH + (hasCheckbox ? CHECKBOXES_WIDTH : 0)) * depth
+  const width = GAP * depth
   return (
-    <span className="dib pr2" style={{ width }}>
-      <span className="flex w-100 justify-end items-center">{children}</span>
-    </span>
+    <>
+      <div className="dib" style={{ width }} />
```

Comment:
Why not use a `span` if the element is inline bloc by css?

Reply:
Because it is a spacer. Even with `span` the `dib` it needed.

Reply:
True, span is inline, not dib.